cli/sawtooth_cli/network_command/compare.py

```diff
@@ -532,28 +586,41 @@ def build_fork_graph(chains, tails):
         the point where they are all at the same block height and the tails of
         the chains from that block height (in the same order).
 
-    Returns a ForkGraph.
+    Returns:
+        A ForkGraph
+        A list of indexes of the chains that had communication problems.
     """
     graph = ForkGraph()
+    bad_chains = []
 
     # Add tails to the graph first
-    for i, tail in enumerate(tails):
+    for i, tail in tails.items():
```

Comment:
Is `i` still an integer value here  or is it now a key?

Reply:
yes to both